### DISCUSSION TITLE
Insta Insanity bug and mod not working properly?

### DISCUSSION THREAD

**Discussion by Hodurlein** [BudgetAirpods/InsanityRemastered#31] [General]
Is there a bug with the mod? When I try solo gameplay, if I turn on the flashlight while in the facility, the insanity automatically goes full, while in no-light spots, it goes to zero when you step on places that have light. I wandered for about 5 minutes with full sanity. Neither a fake player nor other events. I tried adjusting the rng multiplier to 2, but got the same result. I only use these 4 mods: BepInExPack v5.4.2100, DontBlink v1.1.0, InsanityMeter v1.0.0, and InsanityRemastered v1.1.0. **Here's a video evidence link:** https://outplayed.tv/media/KPMBGE

**Comment by BudgetAirpods** [comment]
I just released a hotfix for the instant insanity bug. 

As for the multiplier, it was worded absolutely poorly. It increases it when using something like 2 or 3, and decreases it when using a decimal like 0.5 or 0.3. This has been changed to make it more appropriate. I'm sorry about that one.
